hammers/scripts/clean_old_aggregates.py

```diff
@@ -42,57 +43,53 @@ def aggregates_for_lease(lease):
     return [x for x in aggregates.values() if x['name'] in physical_reservation_ids]
 
 def clear_aggregates(agg_list):
-
     report = []
+    errors = []
 
     for x in agg_list:
         if x['hosts']:
             for host in x['hosts']:
                 try:                
-                    report.append("Deleting host {} from aggregate {} and returning to freepool. ".format(host, x['id']) + "\n")
                     _addremove_host(auth, 'remove_host', x['id'], host)
                     _addremove_host(auth, 'add_host', 1, host)
-                except:
+                    report.append("Deleted host {} from aggregate {} and returned to freepool. ".format(host, x['id']) + "\n")
+                except Exception as exc:
                     report.append("Unexpected error moving host {} from aggregate {} to freepool. ".format(host, x['id']) + "\n")
-                    pass
+                    errors.append(exc)
             try:
-                report.append("Deleting aggregate {}. ".format(x['id']) + "\n")
                 aggregate_delete(auth, x['id'])
-            except:
+                report.append("Deleted aggregate {}. ".format(x['id']) + "\n")
+            except Exception as exc:
                 report.append("Unexpected error deleting aggregate {}. ".format(x['id']) + "\n")
-                pass
-
-    str_report = ''.join(report)
+                errors.append(exc)
 
-    if report != []:
-      return str_report
-    else:
-      return None
+    return errors, report
 
 
 def main(argv=None):
+    slack = Slackbot(args.slack, script_name='clean-old-aggregates') if args.slack else None
```

Comment:
Do we really want one more pep8 violation in hammers just to go down to one line? I'd like refactor hammers at some point to get rid of all these 79>char lines.

Maybe:
`slack = None
if args.slack:
    Slackbot(args.slack, script_name='clean-old-aggregates') `